README.md

```diff
@@ -1 +1,23 @@
-# mobile-version-skeleton
\ No newline at end of file
+project: #Portfolio-mobile-version 
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
```

Comment:
- [ ] Kindly correct this typo error to make your readme more professional. You wrote `crome` instead of `chrome` and remember to **capitalize** your text when starting a new sentence.